### DISCUSSION TITLE
Alignement issue with Outlook Desktop

### DISCUSSION THREAD

**Discussion by DSchuurmanBadge** [bootstrap-email/bootstrap-email#169] [Help]
I am currently working on teh header of my email, in all my email clients it is how it's supposed to look like, except of Outlook Desktop. Can anyone help me? Thanks in advance.

Here is my HTML code:
```html
<!-- Header Begin -->
<div class="w-full p-4 row bg-black">
    <div class="col-lg-9 col-12">
        <!-- Logo of Voetbalshop.nl -->
        <img style="width: 300px;" width="300" src="#">
    </div>
    <div class="col-lg-3 pt-2 pt-lg-0 ax-right">
        <!-- User Icon redirect to login page of Voetbalshop.nl when clicked on -->
        <a class="ax-right no-underline" href="#">
            <img class="contact-icon d-none d-lg-inline" style="width: 25px;" width="25" src="#">
            <p class="text-white d-lg-none">Mijn account</p>
        </a>
    </div>
</div>
<!-- Header End -->
```
-This is how it is supposed to look like:
![Schermafbeelding 2022-03-10 142724](https://user-images.githubusercontent.com/100281297/157672139-732493cc-b8fa-49bf-b39d-6cecf058ef97.png)

-And this is how it looks on Outlook Desktop:
![Schermafbeelding 2022-03-10 142823](https://user-images.githubusercontent.com/100281297/157672188-56750b8a-c525-4e79-b1c4-bd6a69692d06.png)

**Comment by stuyam** [comment]
What do you intend for the contact icon to look like on mobile? Could you just do something like this?
```html
<div>
  <div class="ax-left">Something to the left</div>
  <div class="ax-right">Something to the right</div>
</div>
```
